tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -442,14 +513,14 @@ protected override bool RemoveFast(string key)
         protected override void ClearFast()
         {{            
             MaybeUnknown?.Clear();
-            {(loop.ClassName == "FrameResponseHeaders" ? "_contentLength = null;" : "")}
-            if(FrameHeaders.BitCount(_bits) > 12)
+            ContentLength = null;
+            if(FrameHeaders.BitCount(_bits) > 11)
```

Comment:
Wasn't this just a perf optimization for the case a lot of headers (specifically in `_headers` which now wouldn't include Content-Length) were set? Why would require less headers to be set to take this path now?